src/WebJobs.Script/Dispatch/IProcessRegistry.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.Azure.WebJobs.Script.Dispatch
+{
+    internal interface IProcessRegistry
```

Comment:
>IProcessRegistry [](start = 23, length = 16)

Can we comment this . What is this interface?  What does it mean to Register a process. Why don't we need an unregister.